build.ps1

```diff
@@ -42,7 +42,9 @@ $filesForWindowsPackage = @(
     'RunCommandOnSet.exe',
     'windowspowershell.dsc.resource.json',
     'wmi.dsc.resource.json',
-    'wmi.resource.ps1'
+    'wmi.resource.ps1',
+    'default_settings.v1.dsc.json',
```

Comment:
I'm ok with changing the extension; but the version should stay in the filename otherwise that would complicate update scenario:

The idea was that any new version of dsc package would have default settings files of all previous versions.
E.g. `dsc v3.3` package/installation would have `default_settings.v1.dsc.json`, `default_settings.v2.dsc.json` and `default_settings.v3.dsc.json` (assuming by v3.3 there would be 2 breaking changes in settings schema).

If the filename would be the same across versions, then the proper settings update logic would have to be in the update tool/script (that we don't have yet), rather than in `dsc.exe`.  I though it's easier to have this logic to be in `dsc.exe`.

Reply:
I see.  In that case, my preference would be to have a single json that is essentially a hashtable where the key is the version string.

Reply:
Updated to have a single json file with defaults for all versions.